_studio/mfx_lib/genx/CMakeLists.txt

```diff
@@ -124,19 +124,25 @@ if(MFX_ENABLE_KERNELS)
     ${prefix}/genx_fcopy_gen11lp_isa.cpp
   )
 
+  if(BUILD_KERNELS)
+    add_cmc_target(${MSDK_GENX_ROOT}/h264_encode/src/genx_simple_me.cpp gen8 ${MSDK_GENX_ROOT}/h264_encode/isa)
```

Comment:
compilation will succeed if you write in 2 loops way:
```
foreach(platform gen8 gen9 gen10 gen11 gen11lp)
  add_cmc_target(${MSDK_GENX_ROOT}/h264_encode/src/genx_simple_me.cpp gen8 ${MSDK_GENX_ROOT}/h264_encode/isa)
endforeach()
foreach(platform gen9 gen10 gen11 gen11lp)
  add_cmc_target(${MSDK_GENX_ROOT}/h264_encode/src/genx_simple_me.cpp ${platform} ${MSDK_GENX_ROOT}/h264_encode/isa)
endforeach()
```

So, issue is around passing gen8 as a parameter... hm, "gen8" don't help...

Reply:
There is a bug in add_cmc_target. Fixed by #1113.